docs/Reference/API-Methods.md

```diff
@@ -3854,7 +3854,7 @@ remove a signer.
     methods, use the [`--rpc-http-api`](CLI/CLI-Syntax.md#rpc-http-api) or
     [`--rpc-ws-api`](CLI/CLI-Syntax.md#rpc-ws-api) options.
 
-    The DEBUG API is an more verbose alternative to the [TRACE API](#trace-methods).
+    The DEBUG API is an more verbose alternative to the [TRACE API](#trace-methods) whose main purpose is compatibility with tools such as [Remix](https://remix.ethereum.org/). We recommend using the [TRACE API](#trace-methods) for production use over the DEBUG API. 
```

Comment:
"is **a** more verbose" instead of "is **an** more verbose" ?

Reply:
Yep, good catch!